x3.json

```diff
@@ -0,0 +1,28 @@
+{
+        "response": [
+                {
+                        "maintainer": "Patriot06",
+                        "oem": "Realme",
+                        "device": "Realme X3 Series",
+                        "filename": "crDroidAndroid-13.0-20231001-x3-v9.9.zip",
+                        "download": "https://sourceforge.net/projects/crdroid/files/x3/9.x/crDroidAndroid-13.0-20231001-x3-v9.9.zip/download",
+                        "timestamp": 1696159796,
+                        "md5": "fbbbb15d88881ac1a24ce7db5947569b",
+                        "sha256": "48c5879d61846e66ef666019aae27304399847f1fecf3a4b81c7d76e839645ce",
+                        "size": 889582759,
+                        "version": "9.9",
+                        "buildtype": "Monthly",
+                        "forum": "https://forum.xda-developers.com/t/rom-12-0-rmx2081-crdroid-android-v8-x-unified-official.4432753/",
+                        "gapps": "https://sourceforge.net/projects/flamegapps/files/arm64/canary/android-13/",
+                        "firmware": "",
+                        "modem": "",
+                        "bootloader": "",
+                        "recovery": "https://drive.google.com/uc?id=1LEc6lISIcu5XuaFFrtI0pI_Z3JxjWZwd&export=download",
+                        "paypal": "https://www.paypal.me/patriot235",
+                        "telegram": "https://t.me/patriot_235",
+                        "dt": "https://github.com/crdroidandroid/android_device_realme_x3",
```

Comment:
fix https://github.com/crdroidandroid/android_device_realme_x3/blob/13.0/BoardConfig.mk#L161 as per https://github.com/crdroidandroid/rules-and-guidelines#software